enclave/core/optee/gp.c

```diff
@@ -384,6 +384,10 @@ void oe_abort(void)
 {
     /* No return */
     TEE_Panic(TEE_ERROR_GENERIC);
+
+    /* Ensure that it is unreachable on ARM machines */
```

Comment:
This whole file is arm only. Maybe something like `/* TEE_Panic() does not return, but is not properly annotated. Ensure the compiler does not return from this function */`

Reply:
Now I've got it. I will use your words in a block comment then.